test/passes/optimize-instructions_all-features.txt

```diff
@@ -3422,6 +3434,12 @@
   (drop
    (i32.const 1)
   )
+  (drop
+   (i64.le_u
+    (local.get $y)
+    (i64.const 4294967295)
+   )
+  )
```

Comment:
similar to prev on `binaryen.js` this folded to
```wat
(drop (i32.const 1))
```

So it misinterpretation `i64.const 4294967295` as `i64.const -1`